package.json

```
@@ -59,7 +59,8 @@
     "prettier": "^3.2.5",
     "rimraf": "^5.0.5",
     "turbo": "^1.13.0",
-    "typescript": "~5.3.3"
+    "type-fest": "^4.14.0",
+    "typescript": "~5.4.3"
```

```suggestion
    "typescript": "~5.3.3"
```
ts 最好锁下版本，库兼容性不一定跟得上